src/service/resources-impl.js

```diff
@@ -2016,6 +2040,29 @@ export class ResourcesImpl {
       }
     }
   }
+
+  /**
+   * Listens for scroll events on elements (not the root scroller), and marks
+   * them for invalidating all child layout boxes. This is to support native
+   * scrolling elements outside amp-components.
+   *
+   * @param {!Event} event
+   */
+  scrolled_(event) {
+    const {target} = event;
+    // If the target of the scroll event is an element, that means that element
+    // is an overflow scroller.
+    // If the target is the document itself, that means the native root
+    // scroller (`document.scrollingElement`) did the scrolling.
+    if (target.nodeType !== Node.ELEMENT_NODE) {
```

Comment:
This will only catch a post-iOS13 case. A more general approach is `target !== viewport.getScrollingElement()`

Reply:
Done.